### PR TITLE
{lang}[intel/2015a] Python 3.4.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
wrapper PR for #1598, with updated extensions

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c2e890924596a2d26dc2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2458825df544990e6c17 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e8f99c9c88b39f0d0bc3 for a full test report.

**Comment by boegel** [comment]
OpenSSL easyconfig was added via #1603

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/602adbe88d882ff622aa for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review/test?

**Comment by wpoely86** [comment]
looks fine

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/9d1def2aa04ac5453cb6 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }),
+    ('six', '1.9.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('dateutil', '2.4.2', {
+        'source_tmpl': 'python-%(name)s-%(version)s.tar.gz',
+        'source_urls': ['http://pypi.python.org/packages/source/p/python-dateutil/'],
+    }),
+    ('deap', '1.0.2', {
+        'source_tmpl': '%(name)s-%(version)s.post1.tar.gz',
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: `post1` => `post2`?

Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.4.3-intel-2015a.eb]: fixed, they fixed the issue that required the patch I included (missing `README.md`)

please retest?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ae79edb292905a0dfec9 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/7d47cb08de764d7c7c75 for a full test report.

**Comment by boegel** [comment]
Thanks @wpoely86!

Going in...

**Pull request merged**